src/webots/core/WbApplicationInfo.cpp

```diff
@@ -33,6 +37,24 @@ const WbVersion &WbApplicationInfo::version() {
   return webotsVersion;
 }
 
+const QString &WbApplicationInfo::branch() {
+  static QString version;
+  static bool firstCall = true;
+
+  if (firstCall) {
+    QFile MyFile(WbStandardPaths::webotsHomePath() + "resources/branch.txt");
```

Comment:
That looks better to me:
```suggestion
    QFile branchFile(WbStandardPaths::webotsHomePath() + "resources/branch.txt");
```